### PR TITLE
Zookeepers & pgpool UI 

### PR DESCRIPTION
**Pull request by SayedTahsin**

### CODE REVIEW

File: charts/kubedbcom-pgpool-editor-options/ui/create-ui.yaml [hunk 1]
```diff
@@ -182,6 +142,40 @@ steps:
...
+        schema:
+          $ref: schema#/properties/spec/properties/postgresRef/properties/name
+        type: select
+      label:
+        text: labels.postgresRef.label
+      schema:
+        $ref: schema#/properties/spec/properties/postgresRef
+      show_label: true
+      type: single-step-form
+    - elements:
```
Comment (SamiulSourav) [inline charts/kubedbcom-pgpool-editor-options/ui/create-ui.yaml:169]: use simple switch and fix text as `Sync Users?`

File: charts/kubedbcom-pgpool-editor-options/ui/language.yaml [hunk 2]
```diff
@@ -148,9 +152,8 @@ en:
     province: Province
     provinces: Provinces
     renew_before: Renew Before
-    replicaset:
-      name: Replicaset Name
-      number: Replica Number
+    Cluster:  
+      name: Replicas
```
Comment (SamiulSourav) [inline charts/kubedbcom-pgpool-editor-options/ui/language.yaml:156]: try `replicas: Replicas` and use it in create-ui

File: charts/kubedbcom-zookeeper-editor-options/ui/language.yaml [hunk 3]
```diff
@@ -18,6 +18,8 @@ en:
       replicaset:
         name: Replica Set Name
         number: Replica number
+      cluster:
+        name:
```
Comment (SamiulSourav) [inline charts/kubedbcom-zookeeper-editor-options/ui/language.yaml:22]: remove empty object

File: charts/kubedbcom-zookeeper-editor-options/ui/language.yaml [hunk 4]
```diff
@@ -148,9 +150,8 @@ en:
     province: Province
     provinces: Provinces
     renew_before: Renew Before
-    replicaset:
-      name: Replicaset Name
-      number: Replica Number
+    cluster:
+      name: Replicas
```
Comment (SamiulSourav) [inline charts/kubedbcom-zookeeper-editor-options/ui/language.yaml:154]: same as pgpool

File: charts/kubedbcom-pgpool-editor-options/ui/functions.js [hunk 5]
```diff
@@ -700,6 +722,7 @@ return {
 	showAuthSecretField,
 	showStorageSizeField,
 	getResources,
+  getPostgresList,
```
Comment (SamiulSourav) [inline charts/kubedbcom-pgpool-editor-options/ui/functions.js:725]: can you use tabs instead of spaces?

**Review by SamiulSourav — APPROVED** [review]
LGTM

**Review by nurRiyad — APPROVED** [review]
LGTM

**Pull request merged**